styles/globals.css

```diff
@@ -0,0 +1 @@
+Container {text-align: center;}
```

Comment:
I would avoid writing CSS for React-Bootstrap components since this could overwrite default behavior of the component. Instead, you can assign a className to the <div> component and then write CSS if needed!